src/Microsoft.Identity.Web/Resource/RequiredScopeAttribute.cs

```diff
@@ -23,5 +33,13 @@ public RequiredScopeAttribute(params string[] acceptedScopes)
         {
             Arguments = new object[] { acceptedScopes };
         }
+
+        /// <summary>
+        /// 
```

Comment:
/// <summary>
        /// Default constructor for the attribute. It's meant to be used with the RequiredScopesSettings property.
        /// </summary>
        /// <example><code>[RequiredScope(RequiredScopesSettings="AzureAd:Scopes")]</code></example>

Reply:
fixed. thank you.

---
In reply to: [573118532](https://github.com/AzureAD/microsoft-identity-web/pull/942#discussion_r573118532) [](ancestors = 573118532)